Documentation/git-commit-graph.txt

```diff
@@ -54,6 +54,11 @@ or `--stdin-packs`.)
 With the `--append` option, include all commits that are present in the
 existing commit-graph file.
 +
+With the `--changed-paths` option, compute and write information about the 
+paths changed between a commit and it's first parent. This operation can
+take a while on large repositories. It provides significant performance gains
+for getting file based history logs with `git log`
++
```

Comment:
re: commit-message: RFC Note 1  "we named it `--changed-paths` rather than `--bloom` to capture what...".   (And possibly to later allow for other algorithms??)

Reply:
@jeffhostetler Yeah! We could potentially switch to another data structure that performs better. Dr. stolee found an interesting paper about it today. He probably read the paper, I read the article about the paper :) 

Essentially, the structure in that paper can't be used in our case. But it opens up the possibility of something performing even faster.

Reply:
also, I do worry that the generic term `changed-paths` may get lost in the email-weeds.  Everyone knows and/or has been asking for "Bloom Filters" and it is a well-known term (at least on the mailing list).

Putting "Bloom Filter" in the RFC title and cover letter may help get more traction on the mailing list.

Reply:
@jeffhostetler : Definitely putting blooom filters in the cover letter and the RFC title. I will call out that we have used changed paths for opening the implementation and see how people feel about it.